docs/guides/deployment/health_checks.rst

```diff
@@ -0,0 +1,73 @@
+.. _ref_guide_deployment_health_checks:
+
+=============
+Health Checks
+=============
+
+You may want to monitor the status of your EdgeDB instance. Is it up? Is it
+ready to take queries? This guide will show you to perform health checks using
+our HTTP client and the ``alive`` and ``ready`` endpoints.
+
+Enable the HTTP Client
```

Comment:
`edgeql_http` is _not_ required, the liveness/readiness checks are always available.